src/core/QCheck2.ml

```diff
@@ -698,6 +698,14 @@ module Gen = struct
   let string_size ?(gen = char) (size : int t) : string t =
     bytes_size ~gen size >|= Bytes.unsafe_to_string
 
+  let bytes : bytes t = bytes_size nat
+
+  let bytes_of gen = bytes_size ~gen nat
+
+  let bytes_printable = bytes_size ~gen:printable nat
+
+  let small_bytes ?gen st = bytes_size ?gen small_nat st
```

Comment:
The definition should probably also be updated to be labeled:
```ocaml
let small_bytes ~gen st = ....
```
(and similarly for `small_string`)